### PR TITLE
updated docs: file structure updated, and description added on installation docs page.

### PR DESCRIPTION
**Pull request by irahuldogra**
- Updated folder structure includes components.json on [installation](https://ui.shadcn.com/docs/installation) docs.
- Added components.json description on [installation](https://ui.shadcn.com/docs/installation) docs.

### CODE REVIEW

**Review by dan5py — CHANGES_REQUESTED** [review]
Duplicate of #670

**Comment by irahuldogra** [comment]
@dan5py Seems like the PR was raised at a similar time. Guess, I will update the PR.

File: apps/www/content/docs/installation.mdx [hunk 1]
```diff
@@ -36,6 +36,7 @@ Where is your tailwind.config.js located? › tailwind.config.js
 Configure the import alias for components: › @/components
 Configure the import alias for utils: › @/lib/utils
 Are you using React Server Components? › no / yes
+Write configuration to components.json. Proceed? › (Y/n)
```
Comment (shadcn) [inline apps/www/content/docs/installation.mdx:39]: Let's not put this one here. We only want the prompts and not the confirmation.

File: apps/www/content/docs/installation.mdx [hunk 2]
```diff
@@ -60,6 +61,7 @@ Here's how I structure my app. I use Next.js, but this is not required.
 │   └── utils.ts
 ├── styles
 │   └── globals.css
+├── components.json
```
Comment (shadcn) [inline apps/www/content/docs/installation.mdx:64]: 👍

File: apps/www/content/docs/installation.mdx [hunk 3]
```diff
@@ -71,6 +73,7 @@ Here's how I structure my app. I use Next.js, but this is not required.
 - The rest of the components such as `<PageHeader />` and `<MainNav />` are placed in the `components` folder.
 - The `lib` folder contains all the utility functions. I have a `utils.ts` where I define the `cn` helper.
 - The `styles` folder contains the global CSS.
+- The `components.json` contains the components configuration.
```
Comment (shadcn) [inline apps/www/content/docs/installation.mdx:76]: - The `components.json` contains the components configuration.
- The `components.json` contains configuration for the CLI.

**Review by shadcn — CHANGES_REQUESTED** [review]

File: apps/www/content/docs/installation.mdx (hunk 1 above)
Comment (irahuldogra) [reply apps/www/content/docs/installation.mdx]: Removed it.

**Review by dan5py — APPROVED** [review]

File: apps/www/content/docs/installation.mdx (hunk 2 above)
Comment (irahuldogra) [reply apps/www/content/docs/installation.mdx:63]: 👍

**Pull request closed without merging**